ironfish/src/network/peerNetwork.test.ts

```diff
@@ -290,6 +295,9 @@ describe('PeerNetwork', () => {
           'verifyNewTransaction',
         )
 
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+        expect(node.workerPool.saturated).toEqual(false)
```

Comment:
This feels pretty goofy, but this test wasn't actually testing the case it's supposed to be, so this feels a little better than the test silently changing and not testing in the event of a future refactor